setup.py

```diff
@@ -40,8 +49,9 @@
         "Source": "https://github.com/geopandas/dask-geopandas/",
         "Issue Tracker": "https://github.com/geopandas/dask-geopandas/issues",
     },
-    packages=find_packages(),
+    packages=find_packages(exclude=("tests", "tests.*")),
```

Comment:
I was checking the wheels we created, and noticed that they have a top-level `tests/` directory (which gets installed as a "tests" package in site-packages), which reminded me of https://twitter.com/ocefpaf/status/1466456847125200896 :)

Reply:
I would move `tests` to `dask-geopandas` folder so we ship them with the package. You can then run them from an installed environment which is useful e.g. when verifying if you docker container is properly built.

Reply:
Fine for me as well, moved them inside